.github/workflows/python.yml

```diff
@@ -30,6 +30,7 @@ jobs:
       working-directory: python
       run: |
         sudo apt-get -y -qq install libsnappy-dev
+        sudo apt-get -y install ffmpeg
```

Comment:
should we merge this line with the above line?

Reply:
👍